assets/scss/_variables_project.scss

```diff
@@ -87,9 +90,10 @@ p {
 @media screen and (min-width: 769px) {
```

Comment:
Not really related but I've used [Sass MQ](https://github.com/sass-mq/sass-mq) in the past and it's pretty nice for these types of responsive styling.

Reply:
Thanks! I can add an issue for the contributing community to consider this suggestion in the future. Docsy / Hugo configures most of this of stuff at the theme level in a [separate repo](https://github.com/google/docsy). We would need to override the default behavior, so it would be useful to consider if we need or want to make more fundamental changes to the styling.

I previously overrode `main.scss` when I expanded the footer, but I've since discovered this is causing some odd behavior in Safari.

So related to your comment, https://github.com/imageworks/opencue.io/pull/35 removes the custom `main.scss`, and reverts to the default. After PR is merged, I'm not sure how easy it would be to add Sass MQ.